src/app/components/Image/index.tsx

```diff
@@ -155,16 +157,17 @@ const Image = ({
           </>
         ) : (
           <ImageWrapper>
-            {hasFallback && pageType === FRONT_PAGE && (
-              <>
-                <source srcSet={srcSet} type={mediaType} sizes={sizes} />
-                <source
-                  srcSet={fallbackSrcSet}
-                  type={fallbackMediaType}
-                  sizes={sizes}
-                />
-              </>
-            )}
+            {hasFallback &&
+              (pageType === FRONT_PAGE || pageType === HOME_PAGE) && (
```

Comment:
Let's make this a wee bit easier to read
```suggestion
              ([FRONT_PAGE, HOME_PAGE].includes(pageType) && (
```